### DISCUSSION TITLE
Generate a tessellation with some boundary thickness

### DISCUSSION THREAD

**Discussion by Sofia-Y** [neperfepx/neper#198] [Q&A] [answered]
Hello @rquey,

I am a bit lost as I could not figure out how to generate a tessellation giving the boundaries around the crystals some thickness. Here is the example I am using below. The command (1) runs successfully, however, I don't see any volume fraction created at the boundaries of each grain. 
On another note,  I also want to include some details as shown in command (2), but it keeps aborting. Any help would be very appreciated. Thank you.

(1)  `$ neper -T -n 10 -dim 2 -transform "scale(0.2,0.2)" -domain 'square(0.006,0.006)'`

(2) `$ neper -T -n10 -id 1 -dim 2 -reg 1 -ori unidorm -oridescriptor euler-bunge -transform "scale(0.2,0.2)" -domain 'square(0.006,0.006)'`

**Comment by rquey** [comment, marked as answer]
(1) Cohesive elements have a zero thickness, see a related discussion: #192.

(2) `$ neper -T -n 10 -id 1 -dim 2 -reg 1 -ori uniform -oridescriptor euler-bunge -transform "scale(0.2,0.2)" -domain 'square(0.006,0.006)'`
